### DISCUSSION TITLE
Find a small issue in GitHub interface, and searcing for original repository of GitHub to fix it

### DISCUSSION THREAD

**Discussion by adarshcodes** [community/community#23594] [New to GitHub]
<p>Hey there!<br>
I would like to know the repository of Github’s Site. I wanted to add a minor fix in UI.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>go here,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181078121-cf04dee7-8675-4ed6-b39b-af3406869af2.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84a0d6674dc01c03d0aae84171dc2fa33df59f34_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84a0d6674dc01c03d0aae84171dc2fa33df59f34_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/4/84a0d6674dc01c03d0aae84171dc2fa33df59f34.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84a0d6674dc01c03d0aae84171dc2fa33df59f34_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback" target="_blank" rel="noopener">GitHub - github/feedback: Public feedback discussions for: GitHub for Mobile,...</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - GitHub - github/feedback: Public feedback discussions for: GitH...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>and start a new discussion depending on the category,</p>

**Comment by adarshcodes** [comment]
<p>Thanks for the help, <a class="mention" href="/jdevstatic">@jdevstatic</a></p>
